civihr_default_theme/assets/sass/partials/_navs.scss

```diff
@@ -0,0 +1,23 @@
+// Styles for Navs
+// --------------------------------------------------
+
+.chr_tabs__header {
+    .chr_tabs__header-item {
+        .btn {
```

Comment:
If you're creating this custom component, then define also custom selectors for the `.btn` equivalent, so you don't have to rely on some "external" class (that is, you are now forcing the component to always have a `.btn` inside `.chr_tabs__header-item`).

Why not something like `.chr_tabs__header-item__link`?